### PR TITLE
OpenStack: Set owners for OpenStack templates

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: OWNERS_ALIACES [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
```
Comment (sgreene570) [inline OWNERS_ALIACES]: nit: file name is not correct?

Comment (kikisdeliveryservice) [reply OWNERS_ALIACES]: not a nit, it would be a blocker.

Comment (kikisdeliveryservice) [reply OWNERS_ALIACES]: :)

Comment (Fedosin) [reply OWNERS_ALIACES]: Yeah, my bad :( Thanks for your comment

**Comment by Fedosin** [comment]
@sgreene570 @kikisdeliveryservice folks, what do you think now?

**Pull request merged**